sdk/eventgrid/service.projects

```diff
@@ -19,7 +19,7 @@
 
   <ItemGroup Condition="'$(SDKType)' == 'client'">
     <ProjectsToIncludeBySDKType Include="$(MSBuildThisFileDirectory)Azure.Messaging.EventGrid\**\*.csproj" />
-    <ProjectsToIncludeBySDKType Include="$(MSBuildThisFileDirectory)Microsoft.Azure.EventGrid\**\*.csproj" />
+    <ProjectsToIncludeBySDKType Include="$(MSBuildThisFileDirectory)Microsoft.Azure.EventGrid\**\*.csproj" Condition="'$(AZURE_EVENTGRID_TEST_MODE)' != 'Live'" />
```

Comment:
Can we just split the pipeline and put the T1 package in its own?  Excluding only when it's a live run just seems to muddy things.

Reply:
The goal here was to revert to the state prior to this PR - https://github.com/Azure/azure-sdk-for-net/pull/31927

I can look into just excluding the T1 path directly in the tests.yml.

Reply:
Actually, I don't think there is a way to do it purely in tests.yml - the exclude condition controls what will trigger a pipeline run, not what a pipeline run will actually execute.

Reply:
@weshaggard if you have any suggestions on how to implement this in a better way. I'm trying to get the EG test pipeline passing again and reverting to the previous state of this file.

Reply:
I'd prefer that we split T1 and T2 into separate pipelines entirely.